balance-management/balance-manager/.gitignore

```diff
@@ -0,0 +1,41 @@
+# This file lists build byproducts, 
+# IDE-specific files (unless shared by your team)
+
+
+## Build
+/build
+/dist/
+/target
+**target
+
+## File system
+.DS_Store
+desktop.ini
+
+## Editor
+*.swp
+*.swo
+Session.vim
+.cproject
+.idea
+*.iml
+.vscode
+.project
+.favorites.json
+.settings/
+
+## Temporary files
+*~
+\#*
+\#*\#
+.#*
+
+## Python
+__pycache__/
+*.py[cod]
+*$py.class
+
+## Node
+**node_modules
+**package-lock.json
+
```

Comment:
add keys folder entry to gitignore, because it's keys created while build based on current machine